### DISCUSSION TITLE
Your views on UI

### DISCUSSION THREAD

**Discussion by thekaailashsharma** [WowDrobe/WowDrobe-App#4] [Polls]
What Changes do u feel must be done

**Comment by khyatigupta369** [comment]
Revamp if possible, keeping in mind the cost and efforts

**Reply by prathikshetty2002** [reply]
Most of my friends and community people said that UI looks dope just that we can make some minor changes to it might work well imo

**Comment by thekaailashsharma** [comment]
Sure then let’s find something more interesting
